From 3784da5ea365f576febd8df0f0957cabafd77638 Mon Sep 17 00:00:00 2001
From: Jacob Chen <jacob2.chen@rock-chips.com>
Date: Fri, 6 Jan 2017 18:49:52 +0800
Subject: [PATCH] ARM: configs: rockchip_linux: enable usb-otg related config

This adds select below config for usb-otg:
CONFIG_USB_GADGET_DEBUG_FILES
CONFIG_USB_GADGET_VBUS_DRAW=500
CONFIG_USB_CONFIGFS_UEVENT

Change-Id: Ida3cfc9933125b43a12b75d48b74f81ed95f4f33
Signed-off-by: Jacob Chen <jacob2.chen@rock-chips.com>
---
 arch/arm/configs/rockchip_linux_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/configs/rockchip_linux_defconfig b/arch/arm/configs/rockchip_linux_defconfig
index d9e926800b80..35f482d8649f 100644
--- a/arch/arm/configs/rockchip_linux_defconfig
+++ b/arch/arm/configs/rockchip_linux_defconfig
@@ -350,8 +350,11 @@ CONFIG_USB_SERIAL_QUALCOMM=y
 CONFIG_USB_SERIAL_SIERRAWIRELESS=y
 CONFIG_USB_SERIAL_OPTION=y
 CONFIG_USB_GADGET=y
+CONFIG_USB_GADGET_DEBUG_FILES=y
+CONFIG_USB_GADGET_VBUS_DRAW=500
 CONFIG_USB_CONFIGFS=y
 CONFIG_USB_CONFIGFS_F_FS=y
+CONFIG_USB_CONFIGFS_UEVENT=y
 CONFIG_MMC=y
 CONFIG_MMC_BLOCK_MINORS=16
 CONFIG_MMC_TEST=y
-- 
2.35.3

